styling.css

```diff
@@ -1 +1,446 @@
+* {
```

Comment:
Are you using different output formats in the group? Consider unifying them to the same standard. Is this a generated file?

Reply:
different outputs, planed on fixing it later.